src/lokijs.js

```diff
@@ -804,6 +811,10 @@
             this.autosaveEnable();
           }
         }
+
+        if (this.options.hasOwnProperty('saveThrottled') && this.options.saveThrottled) {
```

Comment:
Why the second check? If saveThrottled is false in options, just set it to this.saveThrottled. No need for another check.